lib/plugins/input/influxHttp.js

```diff
@@ -1,3 +1,4 @@
+/*jshint esversion: 6 */
```

Comment:
I don't think we need jshint version - why is it injected here?

Reply:
Jshint inform me that 
line 6, col 1, 'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).
To skip this warning I added 
/*jshint esversion: 6 */ 
but probably the the right way is add
"esversion"     : 6, to .jshintrc file